index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Addisalem seifu Portofolio</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/e1cc58c746.js" crossorigin="anonymous"></script>
+</head>
+<body>
```

Comment:
![image](https://user-images.githubusercontent.com/43698511/133658375-ee125718-63d2-4dae-9c8e-168b35bec746.png)

- [ ] [OPTIONAL] The Header section and the Hello section you implemented in the first project are messing from the webpage. I highly recommend you to add them as throughout this module you'll be building your portfolio step by step to have a Full page at the end of the module.